.github/workflows/release.yml

```diff
@@ -78,12 +78,12 @@ jobs:
         run: |
           echo '${{ secrets.DOCKER_PASSWORD }}' | docker login -u '${{ secrets.DOCKER_USERNAME }}' --password-stdin
           docker pull cscfi/pyinstaller
-          docker run --rm -v "${PWD}:/builder" cscfi/pyinstaller --onefile sdagui.py --name ${{ matrix.asset_name }}
+          docker run --rm -v "${PWD}:/builder" cscfi/pyinstaller --noconsole --onefile sdagui.py --name ${{ matrix.asset_name }}
           sudo chown -R $USER:$GROUP dist/
       - name: Build GUI artifact
         if: matrix.os != 'ubuntu-latest'
         run: |
-          pyinstaller --onefile sdagui.py --name ${{ matrix.asset_name }}
+          pyinstaller --noconsole --onefile sdagui.py --name ${{ matrix.asset_name }}
```

Comment:
i think here we need to keep --noconsole because of windows bug ? or I am getting it wrong ?

Reply:
it was initially removed as a compromise, but fixed here https://github.com/CSCfi/sda-uploader/pull/38 so we can hide the cmd again for windows

Reply:
using mock to do that, not really ideal, i like this solution explored: https://stackoverflow.com/a/25061573